app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitItemAdapter.java

```diff
@@ -20,16 +21,17 @@
 import java.util.ArrayList;
 import java.util.Locale;
 
-/**
- * The habit item adapter is the custom adapter for the RecyclerView that holds the habit list
- */
 public class HabitItemAdapter extends RecyclerView.Adapter<HabitItemAdapter.HabitViewHolder> {
+    private static final String DATE_PATTERN = "dd-MM-yyyy";
+    private static final Locale LOCALE = Locale.CANADA;
+
     private final Context _context;
     private static HabitList _habits;
-
-    public HabitItemAdapter(Context context, HabitList habits) {
+    private static HabitListFragment _habitListfragment;
+    public HabitItemAdapter(Context context, HabitList habits, HabitListFragment fragment) {
```

Comment:
Minor: we should have a new line break here between the field and the constructor